apps/src/lib/kits/maker/boards/FakeMBBoard.js

```diff
@@ -0,0 +1,208 @@
+/** @file Fake for running Maker apps without an attached MicroBit board. */
```

Comment:
For a bit now, it's seemed like there's been a difference in expectations for the "Fake__Board" classes, where some people were thinking of it as an emulator and others were not. After looking at this file, I think the naming might be what's causing this. It struck me that this is more similar to a Mock that happens to get used in production. I wonder if there's a way we can indicate this in the naming or in the file header comments to resolve some confusion. Edit: After a little googling, I'm realizing Mock might too strongly indicate that this would be used for a test, but I still can't help but wonder if there's a better name for this (or maybe "Fake" is the correct name and I'm just not too familiar with hardware nomenclature).

Similarly if we do end up building an emulator, I wonder if we'd want it to hook into this or into something else.

Reply:
Thanks to you both! I think 'VirtualCPBoard' and 'VirtualMBBoard' work well! @jmkulwik , lmk what you think.

Reply:
Works for me! If it's not too much to ask, ideally, components would follow the same convention. i.e. "VirtualMicroBitButton"

Reply:
Sure - that makes sense.